src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/services/AttendanceServiceTest.kt

```diff
@@ -769,6 +769,34 @@ class AttendanceServiceTest {
                 .putCaseNoteAmendment(1, 1, "IEP rescinded: attended")
     }
 
+    @Test
+    fun `should post case note amendment going from paid attendance to unpaid absent refused`() {
+
+        `when`(attendanceRepository.findById(1))
+                .thenReturn(Optional.of(
+                        Attendance.builder()
+                                .offenderBookingId(1)
+                                .eventLocationId(1)
+                                .eventId(1)
+                                .paid(true)
+                                .attended(true)
+                                .eventDate(today)
+                                .caseNoteId(1)
+                                .build()))
+
+        val service = AttendanceService(attendanceRepository, nomisService)
+
+        service.updateAttendance(1, UpdateAttendanceDto.builder()
+                .attended(false)
+                .paid(false)
+                .absentReason(AbsentReason.UnacceptableAbsence)
+                .comments("Unacceptable absence - No show.")
+                .build())
+
+        verify(nomisService)
+                .putCaseNoteAmendment(1, 1, "IEP reinstated: Unacceptable absence")
+    }
+
     @Test
     fun `should not post a case note amendment going from paid attendance to unpaid absent refused`() {
```

Comment:
This description is saying the case note amendment should 'not' be created, however, the never() has been removed making it inconsistent.

Reply:
The previous test case is still valid, might be worth reverting this change.

Reply:
Ignore the above comments, thought the code below was related to 'should not post a case note amendment going from paid attendance to unpaid absent refused'